privacy/index.md

```diff
@@ -27,8 +27,6 @@ Let’s Encrypt also operates [Community Forums](https://community.letsencrypt.o
 
 Let’s Encrypt may change these policies or providers from time to time. If you have questions about Let’s Encrypt’s privacy practices, please contact them directly at [security@letsencrypt.org](mailto:security@letsencrypt.org).
 
-The Certbot community can be contacted through the Freenode Project’s IRC service, chat.freenode.net, on the #certbot-devel channel. This IRC service is operated by Freenode, which does not publish a privacy policy.
```

Comment:
There appears to be a changelog in this privacy policy ("Changes to This Privacy Poilicy"). Should the removal of this paragraph change be documented there?

Reply:
I can maybe offer a link to the diff change. I will incorporate this feedback today.

Reply:
Apologies on letting this drop off. Added the change